MAINTAINERS.md

```diff
@@ -0,0 +1,15 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
```

Comment:
Should this not be <!-- SPDX-License-Identifier: MPL-2.0 -->

As the project uses this license?

(Also think the copyright contributors might need to change too)

Reply:
Good catch on the copyright statement - fixed now.

For the license, since this is technically documentation it would fall under the documentation license ( CC-BY-40 ).

Reply:
Ah right, I see. Yes we haven't been adding a license to our documentation, but perhaps we should.